web/src/main/java/no/nav/modiapersonoversikt/service/pdl/PdlOppslagServiceImpl.kt

```diff
@@ -79,7 +79,7 @@ class PdlOppslagServiceImpl constructor(
             .data
             ?.hentGeografiskTilknytning
             ?.run {
-                gtBydel ?: gtKommune ?: gtLand
+                gtBydel ?: gtKommune ?: gtLand ?: gtType.toString()
```

Comment:
Vi bør oppdatere funksjon-definisjonen til å reflektere dette. Så slipper vi å gjøre null-sjekker.

Dette vil også returnere strengen "null" om feltene er `null`. Er det det som er tiltenkt her?

Reply:
Funksjonen vil vel fortsatt kunne returnere null siden vi har `?.hentGeografiskTilknytning?.run{}`?

Målet med endringen var egentlig bare å unngå at vi gir `null` her, men en string i stedet (siden `gtType` aldri kan være null). Men det gikk litt fort i svingene fordi vi sender da `gtType` til NORG og det blir jo feil.

Logikken for hvordan NORG håndterer selve gt-string-feltet finner jeg ikke, men vi antar da kanskje at "" vil gi Vikafossen? Egentlig så leser jeg også koden slik at den sjekker først for diskresjonskode, og vil tro at de aldri egentlig leser hva GT faktisk er dersom dette flagget er satt?

Hva om vi sier `gtBydel ?: gtKommune ?: gtLand ?: ""` og så antar at NORG fikser mappingen til Vikafossen?
Vi kan snakke om dette på kaffemøtet 😄